pkg/kademlia/antechamber.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package kademlia
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storage"
+)
+
+// AntechamberErr is the class for all errors pertaining to antechamber operations
+var AntechamberErr = errs.Class("antechamber error")
+
+// Attempts to add a node the antechamber. Only allowed in if within rt neighborhood.
```

Comment:
Should the function starts with `antechamberAddNode` ?

Reply:
I can do that. It's not linter required since it's not an exported method, but if it helps readability, for sure.